#! /bin/sh /usr/share/dpatch/dpatch-run
## 99_ncursesw-include.dpatch by Tobias Grimm <tg@e-tobi.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use ncursesw header file

@DPATCH@
diff -urNad vdr-1.5.16~/PLUGINS/src/skincurses/skincurses.c vdr-1.5.16/PLUGINS/src/skincurses/skincurses.c
--- vdr-1.5.16~/PLUGINS/src/skincurses/skincurses.c	2008-02-23 11:38:04.000000000 +0100
+++ vdr-1.5.16/PLUGINS/src/skincurses/skincurses.c	2008-02-28 00:52:23.000000000 +0100
@@ -6,7 +6,7 @@
  * $Id: skincurses.c 1.22 2008/02/23 10:38:04 kls Exp $
  */
 
-#include <ncurses.h>
+#include <ncursesw/ncurses.h>
 #include <vdr/osd.h>
 #include <vdr/plugin.h>
 #include <vdr/skins.h>
